ichnaea/api/locate/query.py

```diff
@@ -408,3 +426,10 @@ def emit_source_stats(self, source, results):
             "status:%s" % status,
         ]
         self._emit_stat("source", tags)
+        bind_prefix = f'source_{source.name}'
```

Comment:
`emit_source_stats` can be called multiple times for requests with radio data, once with an `internal` source, once with `geoip` source. There's also a `fallback` source, but this code path may not be active anymore.